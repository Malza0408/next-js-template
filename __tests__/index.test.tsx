import { render, screen } from "@testing-library/react";
import Home from "@/pages/index";

describe("Home", () => {
  it("renders a heading", () => {
    const { container } = render(<Home />);
    const heading = screen.getByRole("heading", {
      name: /테스트가 잘 됩니까?/i,
    });

    expect(container).toMatchSnapshot();
    expect(heading).toBeInTheDocument();
  });
});
