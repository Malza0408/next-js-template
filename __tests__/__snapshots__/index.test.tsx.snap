// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders a heading 1`] = `
.emotion-0 {
  color: blue;
}

<div>
  <main>
    <h1
      class="emotion-0"
    >
      테스트가 잘 됩니까?
    </h1>
  </main>
</div>
`;
